R/survival_reg-flexsurv.R

```diff
@@ -27,3 +27,46 @@ flexsurv_rename_time <- function(pred){
       dplyr::rename(.eval_time = .time)
   }
 }
+
+# ------------------------------------------------------------------------------
+# Conversion of quantile predictions to the vctrs format
+
+# For single quantile levels, flexsurv returns a data frame with column
+# ".pred_quantile" and perhaps also ".pred_lower" and ".pred_upper"
+
+# With mutiple quantile levels, flexsurv returns a data frame with a ".pred"
+# column with co.lumns ".quantile" and  ".pred_quantile" and perhaps
+# ".pred_lower" and ".pred_upper"
+
+flexsurv_to_quantile_pred <- function(x, object) {
```

Comment:
There is already another function to use in the `post` hook of the prediction module, called `flexsurv_post()`. Can we call this one `flexsurv_post_quantile()`?